oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -824,15 +835,26 @@ impl WasmNode {
 
 /// Computes the [`NodePrivilege`] granted to a WebAssembly Node running the specified WebAssembly
 /// module.
-fn wasm_node_privilege(wasm_module_bytes: &[u8]) -> NodePrivilege {
+/// Created [`NodePrivilege`] consists of Wasm module hash and signature.
+fn wasm_node_privilege(wasm_module_bytes: &[u8], signatures: Vec<Signature>) -> NodePrivilege {
```

Comment:
It seems this should be able to just get a `&[Signature]` argument, which would be more consistent and also let you avoid the cloning on line 820.

Reply:
I think it's fine to unwrap_or_default in the calling code, but I don't see why this arguments needs to be a Vec, it should be fine with `&[Signature]` and use `.as_ref` on the vec, even for empty vec? https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-AsRef%3C%5BT%5D%3E